src/components/parameters-tabs.tsx

```diff
@@ -377,7 +381,7 @@ const ParametersTabs: FunctionComponent<OwnProps> = (props) => {
                                 label={<FormattedMessage id="VoltageInit" />}
                                 value={TAB_VALUES.voltageInitParamsTabValue}
                             />
-                            <Divider />
+                            <Tab sx={styles.dividerTab} label="" icon={<Divider sx={{ flexGrow: 1 }} />} disabled />
```

Comment:
Maybe add a comment to explain the syntax so we don't reproduce the issue elsewhere ? To make obvious that instantiating a component that's not a Tab directly under a Tabs will induce console warnings